<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>2021 Opencvdl Hw1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox1">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>30</y>
      <width>211</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>1. Image Processing</string>
    </property>
    <widget class="QPushButton" name="Load_Image">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>1.1 Load Image</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Color_Separation">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>1.2 Color Separation</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Color_Transformations">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>1.3 Color Transformations</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Blending">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>1.4 Blending</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>211</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>2. Image Smoothing</string>
    </property>
    <widget class="QPushButton" name="Bilateral_Filter">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>2.2 Bilateral Filter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Median_Filter">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>2.3 Median Filter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Gaussian_Blur">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>2.1 Gaussian Blur</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>211</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>3. Edge Detection</string>
    </property>
    <widget class="QPushButton" name="SobelY">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>3.3 Sobel Y</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SobelX">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>3.2 Sobel X</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Gaussian_Blur_Self">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>3.1 Gaussian Blur</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Magnitude">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>3.4 Magnitude</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>30</y>
      <width>211</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>4. Transformation</string>
    </property>
    <widget class="QPushButton" name="Translation">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>121</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>4.2 Translation</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Rotation_Scaling">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>202</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>4.3 Rotation, Scaling</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Shearing">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>4.4 Shearing</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Resize">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>4.1 Resize</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>932</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
